docx_editor_mcp/tools.py

```
@@ -0,0 +1,824 @@
+    if isinstance(result, dict):
+        return cast(dict[str, Any], result)
+    cached = result
+
+    ext_error = _check_external_changes(cached)
+    if ext_error:
+        return ext_error
+
+    try:
+        change_id = cached.document.insert_after(anchor, text, occurrence=occurrence)
+        cached.mark_dirty()
+        return {"success": True, "change_id": change_id}
```

Ruff (0.15.7)

[warning] Consider moving this statement to an `else` block

(TRY300)